Dockerfile

```diff
@@ -1,30 +1,20 @@
-# using rocker r-vers as a base with R 4.3.1
-# https://hub.docker.com/r/rocker/r-ver
-# https://rocker-project.org/images/versioned/r-ver.html
-
-# set proper base image
-ARG R_VERS="4.3.1"
-FROM rocker/r-ver:$R_VERS AS base
+FROM docker.io/rocker/r-ver:4.3.1 AS base
 
 # set Docker image labels
 LABEL org.opencontainers.image.source=https://github.com/RMI-PACTA/workflow.pacta
-LABEL org.opencontainers.image.description="Docker image to run PACTA"
+LABEL org.opencontainers.image.description="Docker image to run PACTA analysis"
 LABEL org.opencontainers.image.licenses=MIT
-LABEL org.opencontainers.image.title=""
-LABEL org.opencontainers.image.revision=""
-LABEL org.opencontainers.image.version=""
-LABEL org.opencontainers.image.vendor=""
-LABEL org.opencontainers.image.base.name=""
-LABEL org.opencontainers.image.ref.name=""
-LABEL org.opencontainers.image.authors=""
+LABEL org.opencontainers.image.title="workflow.pacta"
+LABEL org.opencontainers.image.vendor="RMI"
+LABEL org.opencontainers.image.base.name="docker.io/rocker/r-ver:4.3.1"
+LABEL org.opencontainers.image.authors="Alex Axthelm, CJ Yetman, Jackson Hoffart"
```

Comment:
just FYI... any of these that are not set to "" will inherit the value set by rocker from the base image

Reply:
`revision` is set by the [Docker Metadata action](https://github.com/docker/metadata-action) that we use in the pipeline, `version` and `ref.name` are being manually overridden [in our build pipeline.](https://github.com/RMI-PACTA/actions/pull/85)